assets/css/_media-queries.scss

```diff
@@ -8,7 +8,7 @@
       box-shadow: 0 0 0.5rem $shadow;
       flex: 3 3 0;
       height: auto;
-      min-width: 180px;
+      min-width: 300px;
       position: relative;
```

Comment:
It does prevent the scroll, but to me that's another issue, as some of the lines are too long. Hiding the horizontal scroll (if it happens to be needed) seems counter-intuitive for me (it's possible to scroll without the wheel, but not everyone knows how). What do you think?